_posts/en/newsletters/2024-03-20-newsletter.md

```diff
@@ -0,0 +1,160 @@
+  automatically defend against).
+
+  Falbesoner's initial work pulls together BIP324 functions written in
+  Python for Bitcoin Core's testing suite, which results in a proxy that
+  is "terribly slow and vulnerable to side-channel attacks [and] not
+  recommended to use it for anything but tests right now".  However, he
+  is working on rewriting the proxy in Rust and may also make some or
+  all of its functions available as a library for light clients or other
+  software that wants to natively support the v2 Bitcoin P2P protocol.
+
+- **Overview of BTC Lisp:** Anthony Towns [posted][towns lisp] to
+  Delving Bitcoin about his experiments over the past couple years
+  creating a variant of the [Lisp][] language for Bitcoin, called BTC
+  Lisp.  See Newsletters [#293][news293 lisp] and [#191][news191 lisp]
+  for previous discussions.  The post goes into significant detail; we
+  encourage anyone interested in the idea to read it directly.  We will
+  briefly quote from its _conclusion_ and _future work_ sections:
+
+  "[BTC Lisp] can be a little expensive on-chain, but it seems like you
+  can do pretty much anything [...] I don’t think implementing either a
+  Lisp interpreter or the bucket of opcodes that would need to accompany
+  it is too hard [but] it is pretty annoying to write Lisp code without
+  a compiler translating from a higher level representation down to the
+  consensus-level opcodes, [though] that seems solvable.  [T]this could
+  be taken further [by] implementing a language like this and deploying
+  it on signet/inquisition."
+
+  Russell O'Connor, developer of the [Simplicity][topic simplicity]
+  language that may also one day be considered as an alternative
+  consensus scripting language, [replied][oconnor lisp] with some
+  comparisons between Bitcoin's current Script language, Simplicity, and
+  Chia/BTC Lisp.  He concludes, "Simplicity and the [Chia Lisp Virtual
+  Machine] clvm are both low level languages that are meant to be easy
+  for machines to evaluate, which causes tradeoffs that make them hard
+  for humans to read. They are intended to be the compiled from some
+  different, human-readable, non-consensus-critical language.
+  Simplicity and the clvm are different ways of expressing the same old
+  things: fetching data from an environment, tupling up bits of data,
+  running conditional statements, and a whole bunch of primitive
+  operations of some sorts.  [...] Since we want this [split between
+  efficient low-level consensus language and high-level non-consensus
+  comprehensible language] regardless, the details of the low-level
+  language become somewhat less important. I.e., with some effort, your
+  high level BTC lisp language could probably be translated/complied to
+  Simplicity [...] Similarly, wherever the design of [Simplicity-based]
+  Simphony [high-level non-consensus language] ends up, it can probably
+  be translated/complied your low level BTC lisp language, with each
```

Comment:
```suggestion
  be translated/complied by your low level BTC lisp language, with each
```
Again, this is within a quotation, so not sure, maybe put the "by" within square brackets?

Reply:
I think this is intended to be "to".  Definitely putting it in square brackets given the ambiguity.